sdk/data_cosmos/src/clients/cosmos.rs

```diff
@@ -154,28 +148,24 @@ fn new_pipeline_from_options(
 }
 
 /// The cloud with which you want to interact.
-///
-/// All variants require the cosmos account name. `Custom` also requires a valid
-/// base URL (e.g. https://custom.documents.azure.com)
-#[allow(unused)]
+// TODO: Other govt clouds?
```

Comment:
Here are the ones we have for Resource Manager endpoints.

https://github.com/Azure/azure-sdk-for-rust/blob/a001d10bd59406d6651cdbc717253a9cb8268f51/sdk/core/src/constants.rs#L1-L14